### PR TITLE
Makes sure HAB_UPDATE_STRATEGY_FREQUENCY_MS is higher than a floor value

### LINKED ISSUE #3363: Constrain use of HAB_UPDATE_STRATEGY_FREQUENCY_MS
This variable determines how often we check for service updates. We should consider constraining its usage somehow (e.g., defining a floor below which you can't set the value).

Setting it to a low value is handy for testing, but you can cause problems if you set it to a very small number in a production setting.

### PR DESCRIPTION
**Pull request by rsertelon**
> Should close #3363

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

**Comment by nellshamrell** [comment]
LGTM

File: components/sup/src/manager/service_updater.rs [hunk 1]
```diff
@@ -311,7 +313,23 @@ impl Periodic for Worker {
...
+                                "{} value ({}) is below the minimal authorized value ({}) \
+                                Falling back to minimal {} MS frequency.",
+                                FREQUENCY_ENVVAR,
+                                num,
+                                MIN_ALLOWED_FREQUENCY,
+                                MIN_ALLOWED_FREQUENCY
+                            );
+                            MIN_ALLOWED_FREQUENCY
+                        }
+                    }
```
Comment (christophermaier) [inline components/sup/src/manager/service_updater.rs]: We should probably handle the (hopefully rare!) case when someone sets a value to 0 or less with the bypass in place.

Comment (rsertelon) [reply components/sup/src/manager/service_updater.rs]: Added a different message in case it's negative ;)

**Comment by christophermaier** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/315547281).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/315547281).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**